content/blog/2016/2016-05-18-external-workspace-manager-plugin.adoc

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "External Workspace Manager Plugin"
+tags:
+- jenkins
+- pipeline
+- plugins
+author: alexsomai
+---
+
+Currently, Jenkins’ build workspace may become very large in size due to the fact that some compilers
+generate very large volumes of data. The existing plugins that share the workspace across builds are able
```

Comment:
Good practice: start each sentence from a new line. It greatly simplifies the maintenance